### PR TITLE
Smashing Magazine Heatmap

### PR DESCRIPTION
**Pull request by ishanchawla1**
This project consists of building a heatmap of the Smashing magazine website

You can find the original project specification at: https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
-# Project Name
+# Smashing Magzine Heatmap
 
-> One paragraph statement about the project.
+> This project was made as a part of Microverse Curriculum for Full Stack Developer for Design-Teardown Project
```
Comment (majovanilla) [inline README.md]: - [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,622 @@
...
+
+.daysAgoFont {
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.noStyle {
+  margin: 0%;
+  padding: 0%;
+}
```
Comment (majovanilla) [inline styles.css]: - [x] No need to include the % when you use 0.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,533 @@
...
+    <main>
+        <nav class="header addBorder">
+            <div class="pageLogo headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Logo</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 1</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 2</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 3</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 4</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 5</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Search</h2></a> </div>
+        </nav>
```
Comment (majovanilla) [inline index.html]: - [x] The structure of your project should have three main parent tags: nav/header, main and footer. Please take the nav tag outside the main tag.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey @ishanchawla1 , 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

Your page has every element it should have and you are making use of the colors to highlight the important elements. It's almost done!

The next changes are minimum, but still necessary for approving your project.

### Required Changes: ✍️

- [ ] Check the comments below.

- [ ] You don't need to include text in all the elements of your page. It would actually look better in terms of the purpose of the project if you could delete some and leave it in one section (the person of the week, for example) Don't delete from the footer.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by thsvr** [comment]
### **STATUS: PROJECT APPROVED** :1st_place_medal: :1st_place_medal: :1st_place_medal: 

Hi @ishanchawla1 and @Zubenna :smile_cat: :smile_cat: 

Congrats for the hard work, your project looks amazing :fireworks: :fireworks: :fireworks: 
You guys implemented all the required changes asked by Maria and your project was approved!

**_Optional Suggestions for the next project:_**

:v:  You can create a feature branch with a more descriptive name on the next projects, like ''main-page'' or ''greyscale-page'' instead of ''feature", for example. It will be more descriptive :smile: 

If you have any questions, don't hesitate to contact me through slack [@Thaís Vieira Azevedo](url)

### **Happy coding and good luck on the next project** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Comment by ishanchawla1** [comment]
Thanks a lot, @majovanilla and @thsvr for the review.

**Pull request merged**